extensions/mssql/src/ui/dialogBase.ts

```diff
@@ -269,28 +271,47 @@ export abstract class DialogBase<DialogResult> {
 		return table;
 	}
 
-	protected addButtonsForTable(table: azdata.TableComponent, addButtonAriaLabel: string, removeButtonAriaLabel: string, addHandler: (button: azdata.ButtonComponent) => Promise<void>, removeHandler: (button: azdata.ButtonComponent) => Promise<void>): azdata.FlexContainer {
+	protected addButtonsForTable(table: azdata.TableComponent, addButtonAriaLabel: string, removeButtonAriaLabel: string, addHandler: (button: azdata.ButtonComponent) => Promise<void>, removeHandler: (button: azdata.ButtonComponent) => Promise<void>, editButtonAriaLabel: string = undefined, editHandler: (button: azdata.ButtonComponent) => Promise<void> = undefined): azdata.FlexContainer {
```

Comment:
Follow up item, but it'd be great to refactor this to have each button be defined as an object that has two properties - aria label and click handler.

Reply:
modified this as suggested and all its references .. Thanks